spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -739,4 +739,61 @@ Data Flow which allows the user to track the tasks launched by the
 NOTE: Releases of `ComposedTaskRunner` can be found
 https://github.com/spring-cloud-task-app-starters/composed-task-runner/releases[here]
 
-
+[[sharing-spring-cloud-dataflows-datastore-with-tasks]]
+== Sharing Spring Cloud Data Flow's Datastore with Tasks
+As discussed in the <<spring-cloud-dataflow-task, Tasks>> documentation Spring
+Cloud Data Flow allows a user to view Spring Cloud Task App executions. So in
+this section we will discuss what is required by a Task Application and Spring
+Cloud Data Flow to share the task execution information.
+
+[a-common-datastore-dependency]
+=== A Common DataStore Dependency
+Spring Cloud Data Flow supports many database <<appendix-migration-guide.adoc#rdbms, types>> out-of-the-box,
+so all the user typically has to do is declare the `spring_datasource_*` environment variables
+to establish what data store Spring Cloud Data Flow will need.
+So whatever database you decide to use for Spring Cloud Data Flow make sure that the your task also
+includes that database dependency in its `pom.xml` or `gradle.build` file.  If the database dependency
+that is used by Spring Cloud Data Flow is not present in the Task Application, the task will fail
+and the task execution will not be recorded.
+
+[a-common-datastore]
+=== A Common Data Store
+Spring Cloud Data Flow and your task application must access the same datastore instance.
+This is so that the task executions recorded by the task application can be read by Spring Cloud Data Flow.
```

Comment:
Perhaps add: "can be read by Spring Cloud Data Flow to list them in the Shell and Dashboard views"